messaging/rust/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "wasmcloud-interface-messaging"
+version = "0.1.1"
+description = "Messaging (wasmcloud:messaging)"
```

Comment:
This description will be on crates dot io, it should include a full sentence that it's a wasmcloud messaging interface.

Reply:
you're right. that's an awful description. will fix.